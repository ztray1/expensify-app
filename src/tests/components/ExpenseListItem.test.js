import React from "react";
import {shallow} from "enzyme";
import expenses from "../fixtures/expenses";
import ExpenListItem from "../../components/ExpenseListItem";

test("should render ExpenListItem f correctly",()=>{
    const wrapper=shallow(<ExpenListItem {...expenses[0]}/>);
    expect(wrapper).toMatchSnapshot();
})