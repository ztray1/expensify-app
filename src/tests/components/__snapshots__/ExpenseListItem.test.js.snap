// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenListItem f correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    $4.44
    -
    December 31st, 1969
  </p>
</div>
`;
